### DISCUSSION TITLE
PU: Error "unauthorized" (Http 401) using authentication with SMC-B certificate

### DISCUSSION THREAD

**Discussion by mge-medisoftware** [Digitaler-Impfnachweis/certification-apis#192] [General]
Beim Übermitteln der signierten challenge und des öffentliches Zertifikates der SMC-B kommt seit heute morgen im Produktivsystem der Http-401-Fehler, der schon im Referenzsystem die letzten Tage zuschlug (dort jetzt aber behoben ist).
Betrifft alle unsere Praxen... (graylog)

**Comment by dirkexe** [comment]
Wir können den Fehler bestätigen. Der Zugriff auf die API ist in allen Arztpraxen ausgefallen (401).

**Comment by mge-medisoftware** [comment]
läuft wieder!

**Comment by dirkexe** [comment]
Mehrere unserer Praxen, die am heutigen Sonntag impfen, melden ein **401 unauthorized**. Der Cert-API-Server (PU) antwortet wirklich mit einem "401 Unauthorized" und einer HTTP-Seite ...

**Reply by mwicke** [reply]
Danke für den Hinweis, das Problem sollte inzwischen behoben sein.

**Reply by dirkexe** [reply]
Danke. Passt. Die Praxen bestätigen positiv: "ging nachmittags wieder"
